extensions/amp-subscriptions/0.1/test/test-crypto-handler.js

```diff
@@ -100,20 +104,28 @@ describes.realWin(
 
       it('should return expected value to a matching key', () => {
         return expect(cryptoHandler.getEncryptedDocumentKey('local')).to.equal(
-          encryptedLocalKey
+          encryptedKey
         );
       });
     });
 
+    describe('decryptDocumentContent_', () => {
+      it('should decrypt the content correctly', () => {
+        return cryptoHandler
+          .decryptDocumentContent_(encryptedContent, decryptedDocKey)
+          .then(decryptedContent => {
+            expect(decryptedContent).to.equal(decryptedContent);
+          });
+      });
+    });
+
     describe('tryToDecryptDocument', () => {
       // eslint-disable-next-line max-len
       it('should replace the encrypted content with decrypted content in multiple sections', () => {
-        return cryptoHandler
-          .tryToDecryptDocument('decryptedDocumentKey')
-          .then(() => {
-            expect(cryptDiv1.textContent).to.equal(' abc ');
-            expect(cryptDiv2.textContent).to.equal(' abc ');
-          });
+        return cryptoHandler.tryToDecryptDocument(decryptedDocKey).then(() => {
```

Comment:
Can we also add a test where we stub the `Subtle.decrypt` method? Some specific arguments are important, such as `iv`, which may not always come through in a pre-canned test.